src/IterableCodeExtractor.php

```diff
@@ -23,12 +23,20 @@ trait IterableCodeExtractor {
 	 * @param array        $options      {
 	 *     Optional. An array of options passed down to static::fromString()
 	 *
-	 *     @type bool $wpExtractTemplates Extract 'Template Name' headers in theme files. Default 'false'.
+	 *     @type bool  $wpExtractTemplates Extract 'Template Name' headers in theme files. Default 'false'.
+	 *     @type array $restrictFileNames  Skip all files which are not included in this array.
 	 * }
 	 * @return null
 	 */
 	public static function fromFile( $file, Translations $translations, array $options = [] ) {
 		foreach ( static::getFiles( $file ) as $f ) {
```

Comment:
Abbreviations like `$f` make the code hard to read. This should have a full name.

The problem seems to be that the proper variable name `$file` is already taken by the other variable, which actually represents "one ore more file or file paths". So either rename the other one so that this one can be called `$file`, or call this one something like `$individual_file`.

Reply:
Done in 119fcf053e3e308532641ba43d82a420b4cae162